### PR TITLE
[IOTDB-1214] Add Flink-IoTDB documents

### PR DESCRIPTION
**Pull request by Sunny-Island**
Add Flink IoTDB documents to website and docs. The documents include English and Chinese version

### CODE REVIEW

File: docs/UserGuide/Ecosystem Integration/Flink-IoTDB.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+# IoTDB-Flink-Connector Example
+
+## Function
+```
+The example is to show how to send data to a IoTDB server from a Flink job.
```
Comment (SteveYurongSu) [inline docs/UserGuide/Ecosystem Integration/Flink-IoTDB.md:26]: Please put the link of the example on the document :D

**Review by SteveYurongSu — CHANGES_REQUESTED** [review]

**Comment by SteveYurongSu** [comment]
Good job. Welcome to the IoTDB community!

**Comment by SteveYurongSu** [comment]
The right doc link: https://github.com/apache/iotdb/tree/master/flink-iotdb-connector

And don't forget the doc of flink-tsfile-connector. :D

File: docs/zh/UserGuide/Ecosystem Integration/Flink IoTDB.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## 使用
+
+* 启动 IoTDB server
+* 运行 `org.apache.iotdb.flink.FlinkIoTDBSink.java` 将 Flink job 运行在本地的集群上。
+
+# TsFile的Flink连接器
+
+## 使用
+
+* 运行 `org.apache.iotdb.flink.FlinkTsFileBatchSource.java` 在本地集群上创建一个 Tsfile 并通过一个 Flink Dataset job 读取它。
```
Comment (jixuan1989) [inline docs/zh/UserGuide/Ecosystem Integration/Flink IoTDB.md:38]: 这些和TsFile相关的似乎不需要在这文档里。

**Review by jixuan1989 — CHANGES_REQUESTED** [review]
see comments

File: docs/UserGuide/Ecosystem Integration/Flink IoTDB.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+# IoTDB-Flink-Connector Example
+
+## Function
+```
+The example is to show how to send data to a IoTDB server from a Flink job.
```
[the remark this answers is not in the document]
Comment (Sunny-Island) [reply docs/UserGuide/Ecosystem Integration/Flink IoTDB.md]: got it

File: docs/UserGuide/Ecosystem Integration/Flink IoTDB.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+# IoTDB-Flink-Connector Example
+
+## Function
```
[the remark this answers is not in the document]
Comment (Sunny-Island) [reply docs/UserGuide/Ecosystem Integration/Flink IoTDB.md]: New doc is available. This change may not be necessary.

**Comment by jixuan1989** [comment]
@Sunny-Island  I modified a few codes.

**Comment by Sunny-Island** [comment]
Look good to me !

**Comment by jixuan1989** [comment]
:( There is an error...
see https://github.com/apache/iotdb/pull/2893

**Pull request merged**